packages/synapse-react-client/package.json

```diff
@@ -79,7 +77,7 @@
     "markdown-it-synapse-heading": "^1.0.1",
     "markdown-it-synapse-math": "^3.0.5",
     "markdown-it-synapse-table": "^1.0.6",
-    "mui-one-time-password-input": "1.1.0",
+    "mui-one-time-password-input": "^2.0.2",
```

Comment:
`mui-one-time-password-input` started causing tests to fail. I am not totally sure why, might be CJS/ESM/TypeScript compatibility issues. Upgrading to 2.0.2 and treating it as an ESM-only package seemed to solve all of the related problems.